src/model.d.ts

```diff
@@ -61,7 +61,7 @@ export interface Transactionable {
   /**
    * The transaction in which this query must be run.
    *
-   * If {@link Options.disableAlsTransactions} has not been set to true, and a transaction is running in the current ALS context,
+   * If {@link Options.disableClsTransactions} has not been set to true, and a transaction is running in the current ALS context,
```

Comment:
```suggestion
   * If {@link Options.disableClsTransactions} has not been set to true, and a transaction is running in the current CLS context,
```
Or `AsyncLocalStorage`

Reply:
Done